components/hab/src/main.rs

```diff
@@ -451,6 +452,13 @@ fn sub_origin_secret_list(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
     command::origin::secret::list::start(ui, &url, &token, &origin)
 }
 
+fn sub_origin_create(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
+    let origin = origin_param_or_env(&m)?;
+    let url = bldr_url_from_matches(&m)?;
```

Comment:
`bldr_url_from_matches` is unfortunately named, especially when viewed next to `auth_token_param_or_env`. It makes it appear that `bldr_url_from_matches ` will *not* look in environment variables for a value, but it will. I'd suggest renaming `bldr_url_from_matches` to have a name more indicative of its behavior. It'll touch a decent number of places, but it should be a find/replace with essentially zero risk.

Reply:
Let's do that separately to minimize any merge conflicts with my PR